OmiseSwift/API Models/Source/SourceType.swift

```diff
@@ -3,9 +3,14 @@ import Foundation
 
 let internetBankingPrefix = "internet_banking_"
 let alipayValue = "alipay"
+let promptPayValue = "promptpay"
+let payNowValue = "paynow"
 let billPaymentPrefix = "bill_payment_"
 let barcodePrefix = "barcode_"
 let installmentPrefix = "installment_"
+let truemoneyValue = "truemoney"
+let payWithPointsValue = "points"
+let payWithPointsCitiValue = "points_citi"
```

Comment:
Why do we have `Pay with Points` two times? I think only the second one is actually working?

Reply:
@tobias90 At first, we can use both of them. For now, after I'm checked with our API team. They already dropped the first one `points` and can only use `points_citi`.
Thanks for a nice catch and I'll remove it in the next PR. 🙇 👍